net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild

```diff
@@ -1,19 +1,17 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=0
+EAPI=7
```

Comment:
This requires a revbump, and a revbump requires resetting the keywords (from stable to ~). Also if you decide to update and revbump, the `dodoc ...` can be replaced with a simple `einstalldocs`.

Reply:
Set back to EAPI 0.